crm/main.py

```diff
@@ -0,0 +1,479 @@
+#!/usr/bin/env python
+
+import click
+import swsssdk
+from tabulate import tabulate
+
+class Crm:
+    def __init__(self):
+        self.cli_mode = None
+        self.addr_family = None
+        self.res_type = None
+
+    def config(self, attr, val):
+        """
+        CRM handler for 'config' CLI commands.
+        """
+        configdb = swsssdk.ConfigDBConnector()
+        configdb.connect()
+
+        configdb.mod_entry("CRM", 'Config', {attr: val})
+
+    def show_summary(self):
+        """
+        CRM Handler to display general information.
+        """
+        configdb = swsssdk.ConfigDBConnector()
+        configdb.connect()
+
+        crm_info = configdb.get_entry('CRM', 'Config')
+
+        print '\nPolling Interval: ' + crm_info['polling_interval'] + ' minute(s)\n'
+
+    def show_thresholds(self, resource):
+        """
+        CRM Handler to display thresholds information.
+        """
+        configdb = swsssdk.ConfigDBConnector()
+        configdb.connect()
+
+        crm_info = configdb.get_entry('CRM', 'Config')
+
+        header = ("Resource Name", "Threshold Type", "Low Threshold", "High Threshold")
+        data = []
+
+        if resource == 'all':
+            for res in ["ipv4_route", "ipv6_route", "ipv4_nexthop", "ipv6_nexthop", "ipv4_neighbor", "ipv6_neighbor",
+                        "nexthop_group_member", "nexthop_group_object", "acl_table", "acl_group", "acl_entry",
+                        "acl_counter", "fdb_entry"]:
+                data.append([res, crm_info[res + "_threshold_type"], crm_info[res + "_low_threshold"], crm_info[res + "_high_threshold"]])
```

Comment:
is crm_info[] from config_db prefixed with "crm" as mentioned in the design spec (CRM_IPV4_ROUTE_THRESHOLD_TYPE)?. Can you place the config_db output for understanding?

Reply:
Currently "CONFIG_DB" output looks as following:
```
127.0.0.1:6379[4]> HGETALL CRM|Config
 1) "ipv4_route_threshold_type"
 2) "percentage"
 3) "ipv4_route_low_threshold"
 4) "70"
 5) "ipv4_route_high_threshold"
 6) "85"
...
```